src/Commands/ClearExpiredImpersonationTokens.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Commands;
+
+use Illuminate\Console\Command;
+use Stancl\Tenancy\Features\UserImpersonation;
+
+/**
+ * This command clears expired impersonation tokens.
+ * By default, all tokens older than UserImpersonation::$ttl (60 seconds by default)
+ * are deleted. To override this, you can use the --ttl option, for example
+ * --ttl=120, all tokens older than 120 seconds will be deleted, ignoring the default.
```

Comment:
Is there a use case for wanting to keep expired tokens?

Reply:
I don't think so